### PR TITLE
[wgsl] Refine pow

### PR DESCRIPTION
**Pull request by axinging**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/5622)

### CODE REVIEW

**Comment by axinging** [comment]
@qjia7  @haoyunfeix @xhcao @gyagp

File: tfjs-backend-webgpu/src/kernels/binary_op_util.ts [hunk 1]
```diff
@@ -208,14 +208,14 @@ const POW_WGSL = `
   if (b == 0.0) {
     return 1.0;
   }
-  if (i32(round(mod(b, 2.0))) != 1) {
+  if (round(abs(b)) % 2.0 != 1.0) {
```
Comment (qjia7) [inline tfjs-backend-webgpu/src/kernels/binary_op_util.ts:211]: Should be `if (round(abs(b) % 2.0) != 1)` if we strictly map to the old code?

Comment (axinging) [reply tfjs-backend-webgpu/src/kernels/binary_op_util.ts]: Done

**Review by qjia7 — APPROVED** [review]
LGTM

**Comment by qjia7** [comment]
cc @lina128 @pyu10055

**Pull request merged**